lib/data_update_scripts/20201223013903_set_proper_display_area_for_profile_fields.rb

```diff
@@ -0,0 +1,10 @@
+module DataUpdateScripts
+  class SetProperDisplayAreaForProfileFields
+    def run
+      ProfileField.where.not(attribute_name: %w[brand_color1 brand_color2]).update_all(display_area: 0)
```

Comment:
**suggestion (non-blocking):** Note that Rails enums are smart, so you can use symbols instead of numbers, which will make this DUS easier to read.

```ruby
ProfileField.update_all(display_area: :header)
#  ProfileField Update All (2.8ms)  UPDATE "profile_fields" SET "display_area" = $1  [["display_area", 0]]
# 29
```

Reply:
👍  this was sort of my first hack at something smaller, but then realized with just a bit more deep work we could probably get this solved more holistically... So will probably take a different direction.